README.md

```diff
@@ -161,4 +174,4 @@ Please see ThirdParty-Licenses directory for details on their licenses.
 ## License and Copyright
 
     OfflineWikipediaTextApi
-    Copyright (C) 2024 Christopher Smith
\ No newline at end of file
+    Copyright (C) 2024 Christopher Smith
```

Comment:
I've been staring at this and for the life of me I can't figure out what changed lol Maybe because of the other additions, github's comparison tool is being silly for the last line?